index.js

```diff
@@ -64,7 +61,7 @@ var new_client = function(api_key, config) {
       this.es.close();
     }
 
-    this.es = new EventSource(this.stream_uri + "/features", {headers: {'Authorization': 'api_key ' + this.api_key}});
+    this.es = new EventSource(this.stream_uri + "/features", {agent: client.proxy_agent, headers: {'Authorization': 'api_key ' + this.api_key}});
```

Comment:
Line is too long.<br>Identifier 'stream_uri' is not in camel case.<br>Mixed double and single quotes.<br>Identifier 'proxy_agent' is not in camel case.<br>Identifier 'api_key' is not in camel case.